agent/engine/daemonmanager/daemon_manager_linux.go

```diff
@@ -111,7 +129,8 @@ func (dm *daemonManager) CreateDaemonTask() (*apitask.Task, error) {
 	// directly, and the object only contains healthcheck, all other fields will be written as empty/nil
 	// in the result string. This will override the configurations that comes with the container image
 	// (CMD for example)
-	rawConfig = fmt.Sprintf("{\"Healthcheck\":%s}", string(rawHealthConfig))
+	rawConfig = fmt.Sprintf("{\"Healthcheck\":%s, \"User\":\"0\"}", string(rawHealthConfig))
+	logger.Info(fmt.Sprintf("Raw Config: %s", rawConfig))
```

Comment:
Can we either make this a DEBUG log or add some more context to it if we want to keep it INFO?